.circleci/config.yml

```diff
@@ -0,0 +1,15 @@
+# https://circleci.com/orbs/registry/orb/circleci/docker
```

Comment:
Should move this to buildkite, no?

Reply:
circleci is very easy for this and thats what I use for docker-symfony, we could move it later :)

Reply:
Just trying to move us off circle quicker. Less time we are on two CI systems, the better